src/nodes.coffee

```diff
@@ -1431,9 +1430,8 @@ exports.Param = class Param extends Base
     return @reference if @reference
     node = @name
     if node.this
-      node = node.properties[0].name
-      if node.value.reserved
-        node = new Literal o.scope.freeVariable node.value
+      name = "at_#{node.properties[0].name.value}"
```

Comment:
I’ve fixed it now. `Scope::freeVariable` can never return a reserved word, because the names it returns always start with an underscore and as of the latest commit there are no reserved words starting with an underscore anymore.

Still, I prefer `@param`s to still compile to `_at_param`, because:

- It is consistent with that all generated variables start with a single `_`, and all helper functions start with two `__` (as said before).
- The not at all unlikely parameters `@base`, `@name`, `@ref` and `@arg` would otherwise be compiled into `_base`, `_name`, `_ref` and `_arg`, which might confuse a reader.
- An extra parameter in `Scope::freeVariable` is needed to force `@a` to compile to `_a1` instead of `_b` if `_a` was already used.
- It is in my opinion more readable, and CoffeeScript should have readable output. I read both `@param` and `_at_param` as “at param”.